cookbook/email/gmail.rst

```diff
@@ -55,33 +50,73 @@ In the development configuration file, change the ``transport`` setting to
             'password'  => 'your_gmail_password',
         ));
 
-You're done!
+If you are using the Symfony Standard Edition, it's more convenient to configure
+these parameters in ``parameters.yml``:
```

Comment:
"these parameters" seems wrong, as we never talked about parameters before in this article. What about: "*If you're using the Symfony Standard Edition, it's more convenient to use parameters for these options:*"